sfdc/client_endpoint.bal

```diff
@@ -557,13 +554,13 @@ public client class Client {
     # + operation - type of operation like insert, delete, etc.
     # + sobj - kind of sobject 
     # + contentType - content type of the job 
-    # + extIdFieldName - field name of the external ID incase of an Upsert operation
+    # + extIdFieldName - field name of the external Id incase of an Upsert operation
     # + return - returns job object or error
     @display {label: "Create job"}
-    isolated remote function creatJob(@display {label: "Operation"} OPERATION operation, 
+    isolated remote function createJob(@display {label: "Operation"} Operation operation, 
```

Comment:
Method name changed due to spelling mistake.

Reply:
Ack